src/Entity/Subscription.php

```diff
@@ -35,11 +35,14 @@
 /**
  * Defines the Subscription entity class.
  *
+ * The label was changed to purchased plan after the entity was created because
+ * it was decided it was a better match.
+ *
  * @\Drupal\apigee_edge\Annotation\EdgeEntityType(
  *   id             = "subscription",
- *   label          = @Translation("Subscription"),
- *   label_singular = @Translation("Subscription"),
- *   label_plural   = @Translation("Subscriptions"),
+ *   label          = @Translation("Purchased Plan"),
+ *   label_singular = @Translation("Purchased Plan"),
+ *   label_plural   = @Translation("Purchased Plans"),
  *   label_count = @PluralTranslation(
  *     singular = "@count Subscription",
```

Comment:
What about the rest?

Reply:
It looked like the other points of #125 were called into question. @cnovak Do we want to use what we have now which results in "Active and Future Purchased Plans", "Cancelled and Expired Purchased Plans", "No active or future purchased plans." and "No cancelled or expired purchased plans." or do we want to switch to hard coded values?

Reply:
Yeah, sorry.... I am utterly confused about the convo we had.  So, best to have @cnovak look at it.

Reply:
Looks good, the purchased plans is now using the entity labels along with the empty list message.  I think you may be concerned that it says "purchased plans" instead of just "plans".  We could modify it, but let's see if we get any feedback before changing.

Reply:
Sort of... I was wondering about the lines directly below these (46+) that remained as "Subscription".  But in #134, [they were all changed](https://github.com/apigee/apigee-m10n-drupal/blob/8.x-1.x/src/Entity/PurchasedPlan.php#L46-L48), so this comment of mine is moot.   Thanks for chiming in here to help clarify! :smile: